indico/MaKaC/PDFinterface/conference.py

```diff
@@ -78,6 +77,30 @@
 ]
 
 
+# Change reportlab default pdf font Helvetica to indico ttf font,
+# because of better support for international characters.
+# Default font can't be easily changed by setting new value to
+# reportlab.rl_config.canvas_basefontname, because this variable
+# is used directly to initialize state of several reportlab modules
+# during their (first) import.
+def getSansStyleSheet():
```

Comment:
So you prefer function name "get_sans_style_sheet"?
(original name imported from reportlab was "getSampleStyleSheet")